packages/terra-framework-docs/src/terra-dev-site/test/table/SortableTable.test.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import Table from 'terra-table';
+import tableData from './mockSortableTableData.json';
+
+const SortableTable = () => {
+  const [tableColumns, setTableColumns] = useState(tableData.cols);
+  const [tableRows, setTableRows] = useState(tableData.rows);
+
+  const handleColumnSelect = (columnId) => {
```

Comment:
You should be using useCallback here for the example.  The way this is coded now, a new method will be created each render, and you will not be able to take advantage of memoization.  We would not want to give that pattern as an example to consumers.

Reply:
Done.